src/content/doc-surrealql/clauses/where.mdx

```diff
@@ -0,0 +1,87 @@
+---
+sidebar_position: 1
+sidebar_label: WHERE
+title: WHERE clause | SurrealQL
+description: The `WHERE` clause can be used to specify a condition that acts as a filter. 
+---
+
+# `WHERE` clause
+
+The `WHERE` clause can be used to specify a condition that acts as a filter. You can use the `WHERE` clause to either filter the result of the FROM clause in a `SELECT` statement or specify which rows to operate on in an `UPDATE`, `MERGE`, or `DELETE` statement.
+
+It can also be used in special cases when working with conditons in [`DEFINE FUNCTION`](/docs/surrealql/statements/define/access) statement or when asserting access control in [`DEFINE TABLE`](/docs/surrealql/statements/define/table) & [`DEFINE FIELD`](/docs/surrealql/statements/define/field) statements.
+
+## Syntax
+
+```syntax title="Clause Syntax"
+STATEMENT
+    [WHERE condition;]
+
+```
+
+## Conditional record selection
+
+The most common use case for the `WHERE` clause is to filter the result of the `SELECT` statement. It is particularly useful when you want to select a subset of records from a table based on a condition.
+
+```surql
+SELECT [FIELDS] FROM [TABLE_NAME] WHERE [CONDITION] = [VALUE];
```

Comment:
How about changing the square bracket syntax like `[FIELDS]` to `@fields`? The square brackets usually mean that something is optional.